pkg/sriov/sriov.go

```diff
@@ -337,6 +340,18 @@ func (s *sriovManager) ApplyVFConfig(conf *sriovtypes.NetConf) error {
 		if conf.SpoofChk == "on" {
 			spoofChk = true
 		}
+		// Save original spoofchk value before override
+		if origVfInfo.Spoofchk {
+			conf.OrigSpoofChk = "on"
+		} else {
+			conf.OrigSpoofChk = "off"
+		}
+
+		if spoofChk && conf.MAC == "" && origVfInfo.Mac.String() == zeroMac {
```

Comment:
I think we may not want to enforce `spoofchk=on + adminMac=all-zero` rule here as this behavior may be different from driver to driver.
I'd suggest to just restore the spoofChk to its original value which shall work across drivers as driver would set the spoofchk and adminMac properly when VF first gets created.

Reply:
It may also be worth documenting this behavior for user reference.

Reply:
Seeing that Intel VFs are created with spoofchk on and all-zero admin mac it seems that behaviour of this state (and transitions to this state) is left to the interpretation of each vendor driver.
will remove the check